source/slurm/overview.rst

```diff
@@ -310,6 +307,36 @@ Pages for a more complete list.
    analysis cluster (gfdl).  On gaea, the ``--clusters`` option must be
    specified, and should be specified for many of the Slurm commands.
 
+Specifying Partitions and QoSes
```

Comment:
All references to "QoS" should be changed to "QOS" for consistency with the rest of this page.

Reply:
Agreed and implemented